app/controllers/account_reset/cancel_controller.rb

```diff
@@ -1,9 +1,24 @@
 module AccountReset
   class CancelController < ApplicationController
+    before_action :check_feature_enabled
+
+    def show
+      render :show and return unless token
```

Comment:
Style: `return render :show unless token` is shorter :golf:

I accidentally did this under a different ticket. I'm uncomfortable relying on the session so much. In my version, I just tacked the token onto the URL for a button that would send a POST. I'm not worried about a browser pre-fetching a POST. But if we want to make sure that someone can't stop an account deletion without going through these steps, then this might be the only way to force it.